### PR TITLE
Add ability to duplicate a question page

### PR DESCRIPTION
**Pull request by samiwel**
### What is the context of this PR?
This PR introduces a new button to the question page toolbar to allow users to duplicate pages via the UI.

Also added some unit tests and a cypress test to cover the duplicate page functionality.

### Dependencies
This PR depends on the following changes:
- https://github.com/ONSdigital/eq-author-api/pull/111
- https://github.com/ONSdigital/eq-author-graphql-schema/pull/57

### How to review 
All tests and checks should pass.
Functionality should work testing against @K9Morphed's branch.

### CODE REVIEW

File: src/components/EditorToolbar/icon-copy.svg [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<svg width="20px" height="20px" viewBox="0 0 20 20" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (hamishtaplin) [inline src/components/EditorToolbar/icon-copy.svg]: This icon should be 32px square, save it in the Sketch file with all the other icons: https://drive.google.com/file/d/11JWbzPatcx92E6ZTnrgOyv73otD9gVCQ/view?usp=sharing

File: src/components/QuestionPageRoute/index.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ import QuestionPageEditor from "components/QuestionPageEditor";
 import IconButtonDelete from "components/IconButtonDelete";
 import { Toolbar, Buttons } from "components/EditorToolbar";
 import IconMove from "components/EditorToolbar/icon-move.svg?inline";
+import iconCopy from "components/EditorToolbar/icon-copy.svg?inline";
```
Comment (hamishtaplin) [inline src/components/QuestionPageRoute/index.js]: `IconCopy` to be consistent with other icons—they are capitalised due to being components.

File: src/components/EditorToolbar/icon-copy.svg (hunk 1 above)
Comment (samiwel) [reply src/components/EditorToolbar/icon-copy.svg]: I've added a 32px width to the project and eyeballed it so it looked about right in terms of surrounding whitespace. I don't seem to have permissions to unlock the sketch file for modification so either you or Kalun may have to add that for me.

Comment (samiwel) [reply src/components/EditorToolbar/icon-copy.svg]: I have managed to save the icon to the sketch file after Kalun gave me write permissions.

**Comment by samiwel** [comment]
The "Copy of" issue should be handled by the API, but I will check this.
